ext/vscode/CHANGELOG.md

```diff
@@ -4,12 +4,17 @@
 
 ### Features Added
 
+- [[3453]](https://github.com/Azure/azure-dev/pull/3453) Adds usage telemetry
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
 
+- [[3467]](https://github.com/Azure/azure-dev/pull/3467) Exposes spinner updates
+- [[3430]](https://github.com/Azure/azure-dev/pull/3430) Exposes provisioned resources for Aspire
+
```

Comment:
These were all VS changes. Not VSCode related. You can remove it probably. Since, it was specifically for VS to integrate with and an implementation detail. cc @weikanglim